core/module/managed_subscribe/src/managed_subscribe_module.rs

```diff
@@ -61,8 +64,9 @@ pub enum TopicAction {
 #[derive(Debug, Deserialize)]
 pub struct ConfigSettings {
     pub base_authority: String,
-    pub managed_subscribe_uri: String,
+    pub managed_subscribe_uri: Option<String>,
```

Comment:
consider using an enum for these new config entries so that it is truly mutually exclusive

Reply:
Updated to an enum, please take a look